tests/test_settings.py

```diff
@@ -0,0 +1,28 @@
+"""
+<Program Name>
+  test_settings.py
+
+<Author>
+  Lukas Puehringer <lukas.puehringer@nyu.edu>
+
+<Started>
+  Jan 30, 2018
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Test in_toto/settings.py
+
+"""
+import unittest
+import in_toto.settings
+
+
+class TestSettings(unittest.TestCase):
+  def test_debug_not_true(self):
+    """in_toto.settings.DEBUG should not be commited with True. """
+    self.assertFalse(in_toto.settings.DEBUG)
+
+if __name__ == "__main__":
+  unittest.main()
```

Comment:
What's the difference between `test_settings` and `test_user_settings`?

Reply:
One tests the `settings` module and the other one tests the `user_settings` module. The latter can be used to configure the former by loading environment variables and rcfiles.

Reply:
> One tests the settings module and the other one tests the user_settings module

I mean that much I could tell. I was imagining both of them are the same functionality, with `user_settings` being just a module/function to override the former and thus needed no conceptual separation. 

I don't think it's a deal at all though.